chain/chunks/src/lib.rs

```diff
@@ -2618,13 +2618,13 @@ mod test {
         let count_forwards_and_requests = |fixture: &ChunkTestFixture| -> (usize, usize) {
             let mut forwards_count = 0;
             let mut requests_count = 0;
-            fixture.mock_network.requests.read().unwrap().iter().for_each(|r| {
+            for r in fixture.mock_network.requests.read().unwrap().iter() {
                 match r.as_network_requests_ref() {
```

Comment:
nit: although more tolerated in closures, can we move away from single letter variables please?  

```suggestion
            for request in fixture.mock_network.requests.read().unwrap().iter() {
                match requst.as_network_requests_ref() {
```